frameioclient/client.py

```diff
@@ -26,12 +31,23 @@ def __init__(self, token, host='https://api.frame.io'):
         status_forcelist=[429],
         method_whitelist=["POST", "OPTIONS", "GET"]
     )
+    self.client_version = self._get_version()
+
+  def _get_version(self):
+    try:
+        from importlib import metadata
+    except ImportError:
+        # Running on pre-3.8 Python; use importlib-metadata package
+        import importlib_metadata as metadata
+
+    return metadata.version('frameioclient')
 
   def _api_call(self, method, endpoint, payload={}):
     url = '{}/v2{}'.format(self.host, endpoint)
 
     headers = {
-      'Authorization': 'Bearer {}'.format(self.token)
+      'Authorization': 'Bearer {}'.format(self.token),
```

Comment:
Is it always given that the token has no `Bearer `? Or does it make sense to double check for it?

Reply:
Not sure I get what you mean here?